data/tribes/immovables/destroyed_building/init.lua

```diff
@@ -7,7 +7,7 @@ descriptions:new_immovable_type {
    -- TRANSLATORS: This is an immovable name used in lists of immovables
    descname = pgettext("immovable", "Destroyed building"),
    icon = dirname .. "menu.png",
-   size = "big",
```

Comment:
:-1:
This change would have a big game-mechanical impact – you cannot build anything directly next to a Big immovable so currently you have to wait until the Destroyed Building turns to Ashes before you can build something new nearby

Reply:
actually the size of this immovable isn't properly taken into account. so I just adjusted the help to reflect real implementation. To verify build a big building in master, destroy it and watch the buildcaps appear of the formerly blocked 3 fields.

Reply:
The size does not have a game-logical connection to the number of fields occupied. The rule "4 nodes for Big, 1 node for every other size" applies only to buildings.

Oh, and workers (and other bobs) can move through immovables with size Small but not through immovables with size Medium or Big, so this too has an impact.

Reply:
well I tested this on crater building a big buioding east of the HQ. after destroying it the big building plot north of  it appeared available, so this immovable appeared to be a small one.   
Regarding the workers I never saw one walking through a small building.

Reply:
This illustrates the difference between a Big immovable and a Small one (images from master):
![grafik](https://user-images.githubusercontent.com/48293446/146671059-75d62519-fe7e-4faa-be9a-04a211f3f1d0.png)
![grafik](https://user-images.githubusercontent.com/48293446/146671067-a85dd17f-b7f5-4e85-9a18-e61d2ac8b0ee.png)


> However I never saw any of them walk through a small building.

I did say below that "[…] buildings are never passable for bobs." :) 
Buildings are a very special case in many regards, and several of the rules that apply to all other immovables do not apply to them and vice versa.

Reply:
ok got it. I just looked above the building and did not realize the change in the right. 
So I'll revert this